examples/leds_shell/README.md

```diff
@@ -0,0 +1,104 @@
+# LEDs and basic GPIO example application
+
+## Description
+
+The application `leds_shell` is a basic example, which allows easy, interactive
+control of internal board LEDs, and basic GPIO for externally connected simple
+devices (for e.g. additional LEDs, relay, motors - via dedicated drivers, etc.).
+
+In particular, this example shows:
+- on/off and toggle internal board LEDs.
+- initialize GPIO port in output mode.
+- set GPIO port state to HIGH or LOW.
+
+## Shell command
+
+The following commands are available:
+
+- `led`: allows switching on/off or toggle internal board LEDs.
+- `init`: call before interact with GPIO port to initialize chosen port in
+  output mode, which allows setting it to HIGH or LOW state.
+- `set`: sets GPIO port state to HIGH.
+- `clear`: sets GPIO port stet to LOW.
+- `help`: default RIOT command, which shows available commands.
+
+## Example on `native`
+
+- Build and run `leds_shell` example application on the `native` target,
+as Linux application:
+
+```
+$ make term
+[...]
+RIOT native interrupts/signals initialized.
+RIOT native board initialized.
+RIOT native hardware initialization complete.
+
+main(): This is RIOT! (Version: 2021.07-devel-10893-gb2e97-example-leds_shell)
+leds_shell, version 1.0.0
+This board has 2 LEDs
+>
+```
+
+Or run it directly without terminal. Go to `RIOT/examples/leds_shell` and run
+commands:
```

Comment:
What's the advantage of this? I'd argue its the same as above, but more verbose and thus more steps for the user.

Maybe just remove this part?